ens/abis.py

```diff
@@ -959,6 +959,43 @@
 ]
 
 RESOLVER = [
```

Comment:
@njgheorghita another place that should get updated eventually to use an EthPM package.  I was under the impression you might already have the ENS contracts packaged up?